bmcremedy_consts.py

```diff
@@ -84,25 +84,27 @@
 BMCREMEDY_JSON_STATUS = "status"
 BMCREMEDY_BLANK_PARAM_ERROR_SUBSTRING = "records have been found for the Incident contact information you have provided"
 BMCREMEDY_CUSTOM_ERROR_MSG = "\nThis can happen if required parameters are not specified in the action.\n"
-BMCREMEDY_ADD_COMMENT_MESSAGE = "Comment added successfully"
-BMCREMEDY_SET_STATUS_MESSAGE = "Set status successful"
+BMCREMEDY_ADD_COMMENT_MSG = "Comment added successfully"
+BMCREMEDY_SET_STATUS_MSG = "Set status successful"
 BMCREMEDY_UPDATE_SUCCESSFUL_MSG = "Incident updated successfully"
 BMCREMEDY_GET_COMMENT_ERROR = "Error while getting comments for incident ID: {id}"
-BMCREMEDY_ERR_INVALID_FIELDS = "Please provide a valid value in the '{field}' parameter"
+BMCREMEDY_ERROR_INVALID_FIELDS = "Please provide a valid value in the '{field}' parameter"
+BMCREMEDY_ENCRYPTION_ERROR = "Error occurred while encrypting the state file"
+BMCREMEDY_DECRYPTION_ERROR = "Error occurred while decrypting the state file"
 
 # Constants relating to 'get_error_message_from_exception'
-ERR_CODE_MSG = "Error code unavailable"
-ERR_MSG_UNAVAILABLE = "Error message unavailable. Please check the asset configuration and|or action parameters."
-PARSE_ERR_MSG = "Unable to parse the error message. Please check the asset configuration and|or action parameters."
+BMCREMEDY_ERROR_CODE_MSG = "Error code unavailable"
+BMCREMEDY_ERROR_UNAVAILABLE_MSG = "Error message unavailable. Please check the asset configuration and|or action parameters."
+BMCREMEDY_PARSE_ERROR_MSG = "Unable to parse the error message. Please check the asset configuration and|or action parameters."
```

Comment:
Would be nice to have ERROR and MESSAGE - or - ERR and MSG instead of word mismatch (not ERROR and MSG)